MvvmCross-Forms/MvvmCross.Forms/Views/MvxFormsListView.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace MvvmCross.Forms.Views
+{
+    public class MvxFormsListView : ListView
+    {
+        public ICommand ItemClick
+        {
+            get;
+            set;
+        } = new Command(() => { });
```

Comment:
Why is there a default value?

Reply:
Testing. There is also no "ItemClick" in the Xaml right now, so this is never going to work. I'm testing why it doesn't work. It's WIP!